src/table/cells/__tests__/handle-cell-focus.spec.js

```diff
@@ -248,4 +248,108 @@ describe('handle-key-press', () => {
       expect(scrollTo).to.have.been.calledOnceWith({ top: targetOffsetTop, behavior: 'smooth' });
     });
   });
+
+  describe('getCellSelectionStatusNote', () => {
+    let rows;
+
+    it('should return singular notation for array with single item', () => {
+      rows = [1];
+      const result = handleCellFocus.getCellSelectionStatusNote(rows);
+      expect(result).to.equal('There is 1 selected value currently');
+    });
+
+    it('should return plural notation for arrays with multiple items', () => {
+      rows = [1, 2, 3];
+      const result = handleCellFocus.getCellSelectionStatusNote(rows);
+      expect(result).to.equal(`There are ${rows.length} selected values currently`);
+    });
+  });
+
+  describe('getCellSrNotation', () => {
+    let selState;
+    let getCellSrNotation;
+
+    beforeEach(() => {
+      selState = { rows: [1, 2] };
+      getCellSrNotation = handleCellFocus.getMemoisedSrNotation();
+    });
+
+    it('should return empty string while we are in first row', () => {
+      focusedCellCoord = [0, 1];
+      const notation = getCellSrNotation({ focusedCellCoord, rootElement, selState });
+
+      expect(notation).to.equal('');
+    });
+
+    it('should return empty string while there is no selected items', () => {
+      selState = { rows: [] };
+      focusedCellCoord = [1, 1];
+      const notatino = getCellSrNotation({ focusedCellCoord, rootElement, selState });
+
+      expect(notatino).to.equal('');
+    });
+
+    it('should return `value selected` and selected rows count while we are selecting a row', () => {
+      focusedCellCoord = [1, 1];
+      selState = { rows: ['some row data'] };
+      const notation = getCellSrNotation({ focusedCellCoord, rootElement, selState });
+
+      expect(notation).to.equal('value is selected. There is 1 selected value currently');
+    });
+
+    it('should return `value selected` and selected rows count while we are selecting multiple rows', () => {
+      const focusedCellCoord = [2, 1];
+      selState = { rows: ['row#01', 'row#02'] };
+      getCellSrNotation = handleCellFocus.getMemoisedSrNotation(1);
+
+      const notation = getCellSrNotation({ focusedCellCoord, rootElement, selState });
+
+      expect(notation).to.equal('value is selected. There are 2 selected values currently');
+    });
```

Comment:
You already have tests for `getCellSelectionStatusNote` so you only need one of these two

Reply:
Yes, you are right, 
Just put it here because of keeping our coverage rate up a bit . if we remove either case, then we might miss related code block in coverage report.

But if you think it's still unnecessary, i can remove one.

Reply:
I don't think it improves coverage. Both of them should run line 109 in handle-cell-focus

Reply:
ok, i'll remove the singular test then

Reply:
✅ Fixed